docs/2_2_common_schema.adoc

```diff
@@ -1032,14 +1081,14 @@ Even if this Dirac impulse would be modeled correctly by the modeling environmen
 Furthermore, in many cases the model equations are derived under the assumption of a constant value (like mass or capacity), and the model equations would be different if "p" would be time varying._
 
 _FMI for Model Exchange:_ +
-_Therefore, "tuning a parameter" during simulation does not mean to "change the parameter online" during simulation.
+_Therefore, "tuning a (structural) parameter" during simulation does not mean to "change the parameter online" during simulation.
 Instead, this is a short hand notation for:_
 
 . _Stop the simulation at an event instant (usually, a step event, in other words, after a successful integration step)._
 
-. _Change the values of the tunable parameters._
+. _Change the values of the tunable (structural) parameters. For tunable structural parameters, the "reconfiguration Mode" state must be entered before and left afterwards._
 
-. _Compute all parameters that depend on the tunable parameters._
+. _Compute all parameters (and sizes of variables, states, derivatives, zero crossings, ...) that depend on the tunable (structural) parameters._
```

Comment:
Maybe we should stick with "event detection" vs. "zero crossing"?

Reply:
right, it should be "event indicators"